seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -147,6 +147,10 @@ bnb,babyswap,1,2022-11-22,0x0e6d9777fc533bc403df3ebe1b82b5399870f1301e45c90427b9
 bnb,babyswap,1,2022-11-30,0x279dfbce106e55f05a20331ed088e1fd2e1ff68c05ab089cca473f3dbbaef9fe,283,0x55d398326f99059ff775485246999027b3197955,1654.599686417120624489,0x53e562b9b7e5e94b81f10e96ee70ad06df3d2657,60085.854
 avalanche_c,trader_joe,1,2022-11-08,0x1f90838d916d2edf881ba53d8abfe5b557ff2a8d23d7ed692cc0687a71a9cd53,15,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,342510510,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,21741961515422881640
 avalanche_c,trader_joe,1,2022-11-09,0x82cd2992be090d073b82406e3f3882c43eeb4ac7e7ce26839290c6e028b2ac3b,130,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,30093631,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,2354558311510002477
+optimism,velodrome,1,2022-12-02,0x8c4641cbf23a94be095e73660510f7e265d76601b414ea6f838f651609741c87,7,0x7f5c764cbc14f9669b88837ca1490cca17c31607,338.699889,0x4200000000000000000000000000000000000042,311
+optimism,velodrome,1,2022-12-02,0x52e9f5710d07923e189104e31b1c98f569a9fa038b41b67043e9b991fcf7a95f,5,0x7f5c764cbc14f9669b88837ca1490cca17c31607,896.242313,0x4200000000000000000000000000000000000042,823.875076815262
+optimism,curve,1,2022-12-02,0x80b901d362d34109fc5ab3d2501390a9a0e54e69388e5eea84469ce77a3b9302,9,0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9,73.0574092246244,0x7f5c764cbc14f9669b88837ca1490cca17c31607,72.2707437014961
+optimism,curve,1,2022-12-02,0x9b98ca1a43c0b4c7709285f8e591a605905dc35e8f52234410628de6ebf1d365,42,0x7f5c764cbc14f9669b88837ca1490cca17c31607,1.034302,0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9,1.03049060456676
```

Comment:
Please have a look on this line.
As the [test query](https://dune.com/queries/1713245?blockchain_t6c1ea=optimism&model_t6c1ea=test_schema.git_28e7e982_curvefi_trades&project_t6c1ea=curve&seed_t6c1ea=test_schema.git_28e7e982_dex_trades_seed&version_t6c1ea=1) showing there may not be matched result in the model.
</head>
<body>


test_description | result (model) | expected (seed) | block_date | blockchain | project | version | tx_hash | evt_index
-- | -- | -- | -- | -- | -- | -- | -- | --
matched records count | 1 | 1 | 2022-12-02 00:00 | optimism | curve | 1 | 0x80b901d362d34109fc5ab3d2501390a9a0e54e69388e5eea84469ce77a3b9302 | 9
matched records count | 0 | 1 | 2022-12-02 00:00 | optimism | curve | 1 | 0x9b98ca1a43c0b4c7709285f8e591a605905dc35e8f52234410628de6ebf1d365 | 42
equality test: token_bought_address | 0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9 | 0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9 | 2022-12-02 00:00 | optimism | curve | 1 | 0x80b901d362d34109fc5ab3d2501390a9a0e54e69388e5eea84469ce77a3b9302 | 9
equality test: token_bought_address |   | 0x7f5c764cbc14f9669b88837ca1490cca17c31607 | 2022-12-02 00:00 | optimism | curve | 1 | 0x9b98ca1a43c0b4c7709285f8e591a605905dc35e8f52234410628de6ebf1d365 | 42
equality test: token_sold_address | 0x7f5c764cbc14f9669b88837ca1490cca17c31607 | 0x7f5c764cbc14f9669b88837ca1490cca17c31607 | 2022-12-02 00:00 | optimism | curve | 1 | 0x80b901d362d34109fc5ab3d2501390a9a0e54e69388e5eea84469ce77a3b9302 | 9
equality test: token_sold_address |   | 0x8c6f28f2f1a3c87f0f938b96d27520d9751ec8d9 | 2022-12-02 00:00 | optimism | curve | 1 | 0x9b98ca1a43c0b4c7709285f8e591a605905dc35e8f52234410628de6ebf1d365 | 42


</body>
</html>

Reply:
> 0x9b98ca1a43c0b4c7709285f8e591a605905dc35e8f52234410628de6ebf1d365

Hm, that's weird. When I copy the compiled code in to Dune, I get those fields populating.
https://dune.com/queries/1725342?d=1 then search for `0x9b98ca1a43c0b4c7709285f8e591a605905dc35e8f52234410628de6ebf1d365`

<img width="539" alt="image" src="https://user-images.githubusercontent.com/4006780/206348894-7cc42f8c-27b7-490e-8310-9e487c065689.png">

Reply:
I think is because of the date, the query result in your model is `2022-12-01`, while in seed is `2022-12-02`.

Reply:
ah yeah I copy and pasted from a csv export for the seed, so that could be manual error. 2022-12-01 is correct